mapbox-gl-native.pro

```diff
@@ -575,3 +574,11 @@ qtConfig(system-zlib) {
 } else {
     QT_PRIVATE += zlib-private
 }
+
+qtConfig(icu) {
```

Comment:
Don't you need to link with ICU when making the shared library? Something like...

`qtbase/src/3rdparty/icu_dependency.pri`